proxy/hdrs/HTTP.cc

```diff
@@ -1558,7 +1558,7 @@ HTTPHdr::_fill_target_cache() const
              (m_host_mime = const_cast<HTTPHdr *>(this)->get_host_port_values(nullptr, &m_host_length, &port_ptr, nullptr))) {
```

Comment:
Could we use the new literal (like port_prt_len) and change the function to get_host_port_values(nullptr, &m_host_length, &port_ptr, &port_prt_len) ?

Reply:
get port_len from ```get_host_port_values()```